core/src/test/scala/anorm/H2Database.scala

```diff
@@ -5,7 +5,7 @@ import scala.util.Random
 
 trait H2Database {
   def withH2Database[R](block: Connection => R): R = {
-    val url = "jdbc:h2:mem:test" + Random.alphanumeric.take(6).mkString("")
+    val url = "jdbc:h2:mem:test" + Random.alphanumeric.take(6).mkString("") + ";DATABASE_TO_UPPER=false"
```

Comment:
I prefer not to change that, to run the integration tests against the default options, and to provide the unit test for this PR using `withLabel` as suggested